Errors When Running the Demo

Currently, the Demo can't be run by following the directions in the README. Hopefully we can either update the readme to reflect the necessary process, or make any fixes needed.

I ran into the following issues, some of which also occurred with the older google-search-results-java:

1. When running the example via `make`, "API_KEY" needs to be capitalized or it will return an error
2. After that, I receive this error: 

```
/bin/sh: ./gradlew: No such file or directory
make: *** [run] Error 127
```
We need to run `make all` or `make init` to initialize the gradle wrapper in the demo folder.

3. Some tests fail during `make dep`:

```
serpapi.BingTest > search FAILED
    java.lang.AssertionError at BingTest.java:34

serpapi.GooglePlayTest > search FAILED
    java.lang.AssertionError at GooglePlayTest.java:36

serpapi.GoogleProductTest > search FAILED
    java.lang.ClassCastException at GoogleProductTest.java:35

serpapi.YahooTest > search FAILED
    java.lang.AssertionError at YahooTest.java:34
```

This occurs regardless of whether the local .jar file or downloading from Jitpack is indicated in `build.gradle`.

4. If I ignore this and continue to `make build`, I run into one of two issues depending on how the dependency is declared in `build.gradle`. 

If `implementation fileTree(dir: "./libs", includes: ['*.jar'])` is uncommented, I run into a similar issue to the one [reported here](https://github.com/serpapi/google-search-results-java/issues/24). 

```
> Task :compileJava FAILED
/Users/schaf/repos/serpapi-java/demo/src/main/java/demo/App.java:14: error: package serpapi does not exist
import serpapi.SerpApi;
              ^
/Users/schaf/repos/serpapi-java/demo/src/main/java/demo/App.java:15: error: package serpapi does not exist
import serpapi.SerpApiException;
              ^
/Users/schaf/repos/serpapi-java/demo/src/main/java/demo/App.java:18: error: cannot find symbol
    public static void main(String[] args) throws SerpApiException {
                                                  ^
  symbol:   class SerpApiException
  location: class App
/Users/schaf/repos/serpapi-java/demo/src/main/java/demo/App.java:34: error: cannot find symbol
        SerpApi serpapi= new SerpApi(parameter);
        ^
  symbol:   class SerpApi
  location: class App
/Users/schaf/repos/serpapi-java/demo/src/main/java/demo/App.java:34: error: cannot find symbol
        SerpApi serpapi= new SerpApi(parameter);
                             ^
  symbol:   class SerpApi
  location: class App
/Users/schaf/repos/serpapi-java/demo/src/main/java/demo/App.java:44: error: cannot find symbol
        } catch (SerpApiException e) {
                 ^
  symbol:   class SerpApiException
  location: class App
6 errors
```
Also, there are some deprecation warnings from Gradle:

```
Deprecated Gradle features were used in this build, making it incompatible with Gradle 9.0.
```

If  `implementation 'com.github.serpapi:serpapi:1.0.0'` is uncommented I get the following error:

```
Execution failed for task ':compileJava'.
> Could not resolve all files for configuration ':compileClasspath'.
   > Could not resolve com.github.serpapi:serpapi:1.0.0.
     Required by:
         project :
      > Could not resolve com.github.serpapi:serpapi:1.0.0.
         > Could not get resource 'https://jitpack.io/com/github/serpapi/serpapi/1.0.0/serpapi-1.0.0.pom'.
            > Could not GET 'https://jitpack.io/com/github/serpapi/serpapi/1.0.0/serpapi-1.0.0.pom'. Received status code 401 from server: Unauthorized
```

--- a/README.md
+++ b/README.md
@@ -39,7 +39,7 @@ We provided a fully working example.
 ```bash
 git clone https://github.com/serpapi/serpapi-java.git
 cd serpapi-java/demo
-make run api_key=<your private key>
+make all API_KEY=<your private key>
 ```
 Note: You need an account with SerpApi to obtain this key from: https://serpapi.com/dashboard
 

